.github/ISSUE_TEMPLATE/1-introduce-open-data-dataset.md

```diff
@@ -0,0 +1,73 @@
+---
+name: Adding External Data to Open Data
+about: ArcGIS Online items from other organizations
+title: Add <dataset name> from <source> to Open Data
+labels: "introduction, porter"
+assignees: "@steveoh, @gregbunce, @rkelson"
+---
+
+# Summary
+
+_Please introduce your data, where it comes from, why it is being added etc._
+
+- **Dataset Name**: Utah FooBar
+- **ArcGIS Online Item Link**: <orgname>.maps.arcgis.com/home/item.html?id=<itemid>
+- **ArcGIS Online user name**: glob
+- **Proposed Open Data [Group](https://utah.maps.arcgis.com/home/groups.html)**: Quux
+- **Data Description**: Bar
+- **Source Agency/Entity**: Baz
+- **Dataset Contact Name**: Bar Baz
+- **Dataset Public Contact Number and/or Email**: foo@bar
+- **Update Frequency**: Quarterly
+
+## Submitter's Process
+
+1. _Please review our [Open Data sharing process](https://gis.utah.gov/sharing/process#open-data) and [qualifications](https://gis.utah.gov/about/policy/sgid#open-data-qualifications)._
+1. _Request access to the [Open Data Staging Group](https://utah.maps.arcgis.com/home/group.html?id=3d3bd0d238f24f45b2c4b84f1015a317) if you don't have access already._
+1. _Check [x] the box in the items below when you have completed the task and add the date of completion._
+
+### Initial Submission
+
+- [ ] Provide all the summary info above (`2021/01/01`)
+- [ ] Share the ArcGIS Online item with our [Open Data Staging Group](https://utah.maps.arcgis.com/home/group.html?id=3d3bd0d238f24f45b2c4b84f1015a317) (`2021/00/00`)
+
+### Post-Review
+
+1. _Complete these after you've been notified that the dataset is ready to be added to Open Data_
+
+- [ ] Remove the item from the staging group (name, completed: `2021/00/00`)
```

Comment:
**question**: If these two items are meant for agrc to do then we should say that so the issue creator knows where to stop or what to ignore. Otherwise we should probably remove the `name` part.

Reply:
Good catch, I meant to remove `name`.